bitnami/minio/values.yaml

```
@@ -224,6 +224,7 @@ securityContext:
   enabled: true
   fsGroup: 1001
   runAsUser: 1001
+  runAsNonRoot: false
```

See above description
```suggestion
  runAsNonRoot: true
```